AlfrescoApp/View Controllers/Settings View Controller/SettingsViewController.m

```diff
@@ -448,7 +448,11 @@ - (NSString *)tableView:(UITableView *)tableView titleForFooterInSection:(NSInte
     NSString *groupFooterTitle = [groupInfoDictionary objectForKey:kSettingsGroupFooterLocalizedKey];
     
     // TODO: Find a cleaner way to replace this section footer
-    if ([groupFooterTitle isEqualToString:@"settings.send.diagnostics.description"])
+    if ([groupFooterTitle isEqualToString:@"settings.security.passcode.lock.description"])
```

Comment:
Future improvement: It seems like here would actually be a good place to use string constants!